jujugui/static/gui/src/app/components/search-results/test-search-results.js

```diff
@@ -623,6 +623,35 @@ describe('SearchResults', function() {
         promulgated: true,
         id: 'mysql',
         type: 'charm'
+      }, {
+        name: 'wordpress',
+        displayName: 'wordpress',
+        url: 'http://example.com/wordpress',
+        downloads: 1000,
+        owner: 'charmers',
+        promulgated: true,
+        id: '~charmers/wordpress',
```

Comment:
Can we use real id's for these? This is kind of why we ended up with this bug: 

"cs:bundle/mediawiki-single-9"

"cs:trusty/rabbitmq-server-43"

for example